src/protocol/thread/thread.ts

```diff
@@ -294,7 +294,7 @@ class _ThreadFront {
 
     client.Session.addLoadedRegionsListener((parameters: loadedRegions) => {
       // Log loaded regions to help with diagnostics.
-      console.log("LoadedRegions", parameters);
+      // console.log("LoadedRegions", parameters);
```

Comment:
Let's keep this in (for now): https://github.com/RecordReplay/devtools/commit/7b961a6d6f258c5a0ec0e8fc2a67ceb8a699f142